ofrak_core/ofrak/core/complex_block.py

```diff
@@ -89,7 +93,7 @@ async def get_mode(self) -> InstructionSetMode:
         :raises ValueError: if the basic blocks in the complex block have more than one mode
         :return: the mode of the complex block
         """
-        await self.resource.unpack()
+        await self.resource.auto_run(all_unpackers=True)
         bb_modes = {bb.mode for bb in await self.get_basic_blocks()}
         if len(bb_modes) == 1:
             return bb_modes.pop()
```

Comment:
I think this point is worth discussing, but perhaps beyond the scope of this PR. Can we make an issue for it?

Reply:
I was thinking the same thing when I went back to this code, but definitely beyond the scope of this MR. The change made to this function was just to limit the scope of the auto-run to unpackers, not identifiers (not necessary as the resource here has already necessarily been tagged as a ComplexBlock, and not desirable because it created an analysis loop/deadlock when running the linkable symbol identifier).